src/components/OrderList/OrderListItem.tsx

```diff
@@ -1,41 +1,78 @@
+import {observer} from 'mobx-react';
 import * as React from 'react';
 import {OrderModel, Side} from '../../models';
+import {toLocaleStringWithAccuracy} from '../../utils/string';
 import {Icon} from '../Icon/index';
-import {OrderActions, OrderListProps} from './index';
+import {Cell} from '../Table/styles';
+import {OrderActions, OrderCellWidth} from './index';
 
-const OrderListItem: React.SFC<OrderModel & OrderActions & OrderListProps> = ({
-  createdAt,
-  cancelOrder,
-  price,
-  id,
-  side,
-  symbol,
-  volume,
-  onEdit
-}) => {
-  const colorSide = side === Side.Buy ? '#fb8f01' : '#d070ff';
-  return (
-    <tr>
-      <td>{symbol}</td>
-      <td>
-        {/* tslint:disable-next-line:jsx-no-lambda */}
-        <span onClick={() => cancelOrder!(id)}>
-          <Icon name="cross" />
-        </span>
-      </td>
-      <td>{id}</td>
-      <td style={{color: colorSide}}>{side}</td>
-      <td>{volume}</td>
-      <td>{price}</td>
-      <td>{createdAt.toLocaleString()}</td>
-      <td>
-        {/* tslint:disable-next-line:jsx-no-lambda */}
-        <span onClick={() => onEdit(id)}>
-          <Icon name="pencil" />
-        </span>
-      </td>
-    </tr>
-  );
+interface OrderListItemProps {
+  onEdit: any;
+  order: OrderModel;
+  accuracy: number;
+  symbol: string;
+}
+
+const getFilled = (
+  volume: number,
+  remainingVolume: number,
+  accuracy: number
+) => {
+  return toLocaleStringWithAccuracy(volume - remainingVolume, accuracy);
+};
+
+const getFilledPercent = (
+  volume: number,
+  remainingVolume: number,
+  accuracy: number
+) => {
+  return volume - remainingVolume === 0
+    ? 0
+    : toLocaleStringWithAccuracy(remainingVolume / volume * 100, accuracy);
 };
 
+const OrderListItem: React.SFC<OrderActions & OrderListItemProps> = observer(
```

Comment:
plz make it`observer` in the index or in the default export, at least. the component itself is agnostic to the state management solution